solutions/src/test/java/ru/sberbank/school/task08/SerializationTest.java

```diff
@@ -0,0 +1,66 @@
+package ru.sberbank.school.task08;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import ru.sberbank.school.task08.state.GameObject;
+import ru.sberbank.school.task08.state.InstantiatableEntity;
+import ru.sberbank.school.task08.state.MapState;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Created by Mart
+ * 03.06.2019
+ **/
+public class SerializationTest {
+    private String fileName = "saved.tmp";
+    private MapState<GameObject> saved;
+    private String directoryPath = "D:\\";
+
+    @BeforeEach
+    void init() {
+            List<GameObject> list = new ArrayList<>();
```

Comment:
&#39;method def&#39; child has incorrect indentation level 12, expected level should be 8.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1445072734>*